_content/av/grumpy-old-monks_sujato.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Grumpy Old Monks"
+authors:
+  - sujato
+external_url: "https://www.youtube.com/watch?v=rPAX-zkE3aQ"
+drive_links:
+  - "https://drive.google.com/file/d/1qCHxDEVBrQSGhFkwVnv8cu5D8GZRcw59/view?usp=drive_link"
+formats: [mp3]
+course: thai
+tags:
+  - wider
```

Comment:
To be tagged with "wider" it should be about the material history of all humanity. While I personally think the world of Ajahn Mun and Luangta Mahabua, I don't know how many humans would agree! 🤭 

Maybe let's go with `modern` instead. 🤔 Maybe `form`?